### PR TITLE
Added boilerplate.

### PR DESCRIPTION
**Pull request by beinginorbit**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+</head>
+<body>
+  <div class="container">
+    <div class="row">
+      <div class="col-12">
+        <h1>SMU Bootcamp Students</h1>
+
+        <h2>Instructor Favorite Websites</h2>
+        <ul>
+          <li>Google</li>
```
Comment (beinginorbit) [inline index.html]: Looks like you forgot to link this.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+  <title>Document</title>
```
Comment (rainczech) [inline index.html]: Page needs a title, plz

Comment (beinginorbit) [reply index.html]: Got it! Thanks!

**Review by rainczech — CHANGES_REQUESTED** [review]
your code is bad.

**Review by rainczech — APPROVED** [review]
Looks good. Marking Complete.

**Pull request merged**